kin-devplatform-sdk/src/main/java/kin/devplatform/data/blockchain/BlockchainSourceImpl.java

```diff
@@ -398,13 +399,15 @@ public KeyStoreProvider getKeyStoreProvider() {
 
 	@Override
 	public void updateActiveAccount(int accountIndex) throws BlockchainException {
-		local.setAccountIndex(accountIndex);
+			local.setAccountIndex(accountIndex);
 		createKinAccountIfNeeded();
 
-		balanceRegistration.remove();
-		startBalanceListener();
-		//trigger balance update
-		getBalance(null);
+			if (balanceRegistration != null) {
+				balanceRegistration.remove();
+			}
+			startBalanceListener();
+			//trigger balance update
+			getBalance(null);
```

Comment:
Same comment as above - format the code because i think you have an extra tab

Reply:
:+1: